README.md

```diff
@@ -145,6 +145,8 @@ dotnet run --project LeaderboardBackend --urls https://localhost:7128 // dotnet
 The value provided to `--urls` has to match what's listed under `applicationUrl` in `LeaderboardBackend/Properties/launchSettings.json`. As of this writing, it's `https://localhost:7128`.
 
 #### Test the App
+Docker is required to run integration tests against a real database.  
+However, if you would like to run the tests with an in-memory database, you can set the following environment variable: `INTEGRATION_TESTS_DB_BACKEND=InMemory`.
```

Comment:
Since you've mentioned that we can have situations where tests can work if we use an in-memory DB, but fail in a real one, do we want to mention that here? Maybe encourage contributors to always test with a real DB instance before requesting review?